org.eclipse.jdt.ls.core/schema/org.eclipse.jdt.ls.core.participant.exsd

```diff
@@ -0,0 +1,112 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="org.eclipse.jdt.ls.core" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="org.eclipse.jdt.ls.core" id="org.eclipse.jdt.ls.core.participant" name="lsparticipant"/>
+      </appinfo>
+      <documentation>
+         This extension point allows to add a participant to offer commands for language server.

```

Comment:
This extension point allows to add a delegate command handler to extend the Java Language server